app/views/layouts/application.html.erb

```diff
@@ -45,7 +45,7 @@
             <button type="submit" class="btn btn-default">Submit</button>
           </form>
           <ul class="nav navbar-nav navbar-right">
-            <li><a href="#">Log Out</a></li>
+            <li> <%= link_to "Log out", session_path, method: :delete %>.</li>
```

Comment:
Awesome. Next step would be to not have the Log Out link to always be showing, instead it would be in a conditional with the Sing Up link.